### ISSUE TITLE
authblk package results in inconsistent author lining

### ISSUE THREAD

**Issue by SSoelvsten** [SSoelvsten/latex-preamble-and-examples#22]
When creating multiple authors as follows
```latex
\author{Anders A. 
\\Bo B.
\\Claus C.
}
```
Then there is an unepxected gap between _Anders A._, _Bo B._ og, _Claus C._. This is due to importing `authblk` in `preamble_base_1.tex`

The intended usage is
```latex
\author{Anders A, Bo B., Claus C.}
```
which works, so I'll put this under `wontfix`

**Comment by SSoelvsten** [comment]
Since _authblk_ also breaks the beamer package. Maybe I really should remove it, since I don't use footnote affiliation anyway.

**Comment by SSoelvsten** [comment]
Closed with 3aafccd0be53e2a06c3891c3bb78a0c022f95826
